sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -158,12 +158,14 @@ case class CreateViewCommand(
       def verify(child: LogicalPlan) {
         child.collect {
           // Disallow creating permanent views based on temporary views.
-          case UnresolvedRelation(AsTableIdentifier(ident))
```

Comment:
nit: `import sparkSession.sessionState.analyzer.AsTableIdentifier` can be removed.